src/test/java/com/google/api/codegen/gapic/testdata/py/python_library.baseline

```diff
@@ -4567,8 +4567,10 @@ from google.cloud.example import library_v1
 import six
 
 def sample_get_big_book(shelf, big_book_name):
-  """Testing resource name overlap"""
-
+  """ Testing resource name overlap
+  Args:
+  big_book_name: The name of the book.
```

Comment:
How is line wrapping handled? Could you add a longer description to one of the params to test wrapping > 80 chars?

Reply:
Line wrapping is not handled :( I _think_ artman format can handle it correctly? Actually let me test it real quick.

Reply:
Hmmmmm, clearly I am wrong :( Only Java formats the long line correctly. I'm gonna check with artman experts, and if that doesn't work out we may need to do it in gapic-generator.